_posts/2019/2019-12-17-what-working-with-a-distributed-team-taught-us-about-agile-team-collaboration.md

```diff
@@ -0,0 +1,80 @@
+We could see all of the team members individually, could better get their mood by reading their face expressions and most importantly, I could see when someone unmuted, because she wanted to say something and react to that. 
+In the conference room setting, it was difficult to impossible for me to get those small signals. 
+At the same time, the team brought up the idea of a mini retrospective at the end of every meeting. 
+It took just five minutes and provided really valuable input for me. 
+The team members gave feedback on whether they like or disliked the new online tools we used, whether they wanted more or less strict moderation, and especially also gave feedback on how the sound quality was.
+
+## Improve your sound quality
+
+Everyone who works with distributed teams knows about this and I can already hear your sighs. 
+We had all the issues from bad microphone quality (so you could barely hear what your colleagues were saying) to Google Hangouts  automatically levelling the sound level of your microphones. 
+This was and still is my least favourite part about working in a distributed team, but the time spent was definitely worth it. 
+We kept looking for solutions until we had fixed all technical issues. 
+Annoying, but it worked. 
+I had even set up a meeting with my PO, who is working in a different location that I am, just to try out different devices for  45 minutes. 
+We also tried out things, that could affect sound quality (e.g. a laptop between you and the microphone), in order to make the teams sensible to what to watch out for. 
+It definitely helped to be in the remote position from time to time to experience the limiting factors yourself. 
+Another option is to record your meeting (e.g. the Standup) to become aware of sound quality issues and common patterns.
+
+## Remote first as a principle
+
+Early in our learning path towards a better remote collaboration, one of our developers recommend a podcast with Bryan Helming, Co-Founder of Zapir, a company that works exclusively remote. In this podcast, Bryan said, one of the company’s values they committed to was “Remote first”. 
+Like I had said in the beginning, examine every process on how you could make a physical process work for a distributed team.
+I don’t want to bore you with the details, but besides banning all posters or physical processes we had in the team office, we also emphasised the already existing culture of using the team chat for all work related discussions and documenting all decisions in our Confluence space. 
+It also helps to consciously change your habit of just asking your closest colleague of the team to just picking someone from another location to ask for an advice or his opinion on a certain topic. 
+Besides using Retrium for the retrospective , we also looked for a tool to make estimation more smoothly. 
+We chose www.planitpoker.de because it easily allows everyone to estimat and gives a nice visual overview of the outcome. 
```

Comment:
Typo: estimat => estimate
Wrong URL: planitpoker.de => planitpoker.com

```suggestion
We chose [planITpoker](https://www.planitpoker.com) because it easily allows everyone to estimate and gives a nice visual overview of the outcome. 
```